pandas/io/tests/test_stata.py

```diff
@@ -780,8 +780,14 @@ def test_drop_column(self):
         expected = expected[columns]
         dropped = read_stata(self.dta15_117, convert_dates=True,
                              columns=columns)
-
         tm.assert_frame_equal(expected, dropped)
+
+        columns = ['int_', 'long_', 'byte_']
+        expected = expected[columns]
+        reordered = read_stata(self.dta15_117, convert_dates=True,
+                               columns=columns)
```

Comment:
can you tests with an invalid column? (not exactly sure what we should do though), could reindex, ``KeyError`` or ignore.